.husky/pre-commit

```diff
@@ -1 +1,2 @@
+node scripts/lint-staged-files.mjs
```

Comment:
On pre-commit, now run this script, which will lints the affected projects (instead of linting the entire monorepo).